pypact/input/inputdata.py

```diff
@@ -458,3 +468,44 @@ def _deserialize(self, f):
                 if len(parts) != 2:
                     raise PypactInvalidOptionException("Invalid DENSITY line format.")
                 self.setDensity(float(parts[1]))
+
+            elif line.startswith("FLUX") and irradiation_active:
+                # Parse the FLUX line
+                parts = line.split()
+                if len(parts) != 2:
+                    raise PypactInvalidOptionException("Invalid FLUX line format.")
+                fluxAmp = float(parts[1])
+
+                # Ensure FLUX is set to a non-zero value before the first irradiation step
+                if not flux_set and fluxAmp <= 0.0:
```

Comment:
I think you can do just decay in F-II without any `ZERO` keyword, but this needs to be confirmed.